src/components/tl/mlx5/tl_mlx5_team.c

```diff
@@ -117,52 +113,45 @@ ucc_status_t ucc_tl_mlx5_team_create_test(ucc_base_team_t *team)
     switch (tl_team->state) {
     case TL_MLX5_TEAM_STATE_INIT:
         status = ucc_service_allreduce(
-                    core_team, &tl_team->status[0], &tl_team->status[1],
-                    UCC_DT_INT32, 1, UCC_OP_MIN, subset, &tl_team->scoll_req);
+            core_team, &tl_team->a2a_status.local, &tl_team->a2a_status.global,
```

Comment:
this service allreduce post should be moved to team init

Reply:
I think it's not possible since service team is not initialized at team init